pkg/volume/csi/csi_mounter.go

```diff
@@ -278,7 +279,7 @@ func (c *csiMountMgr) SetUpAt(dir string, mounterArgs volume.MounterArgs) error
 	}
 
 	if c.supportsFSGroup(fsType, mounterArgs.FsGroup, c.fsGroupPolicy) {
-		err := volume.SetVolumeOwnership(c, mounterArgs.FsGroup, mounterArgs.FSGroupChangePolicy)
+		err := volume.SetVolumeOwnership(c, mounterArgs.FsGroup, mounterArgs.FSGroupChangePolicy, util.FSGroupCompleteHook(c.plugin.GetPluginName()))
```

Comment:
should the plugin name be "csi", or should it be the name of the csi driver?

Reply:
+1, it should probably be c.driverName or maybe pluginName/driverName

Reply:
ohh, sorry! I will take other PR to fix it.  Thanks for pointing at it!

Reply:
+1 to capturing full plugin name. we should use `GetFullQualifiedPluginNameForVolume` I think.

Reply:
IMO,  c.driverName is more detailed maybe.

https://github.com/kubernetes/kubernetes/blob/87cc9bca0fcbbdf1b8232c9c71e50023d62667c3/pkg/volume/csi/csi_plugin.go#L357-L370 

c.driverName gets driver name from `CSIVolumeSource` and `CSIPersistentVolumeSource` 

While, `GetFullQualifiedPluginNameForVolume` get driver name from `CSIPersistentVolumeSource`

https://github.com/kubernetes/kubernetes/blob/87cc9bca0fcbbdf1b8232c9c71e50023d62667c3/pkg/volume/util/metrics.go#L122-L127

See https://github.com/kubernetes/kubernetes/pull/95892